src/pages/workspaces/workspace/delete-workspace-modal.state.ts

```diff
@@ -0,0 +1,111 @@
+import _ from 'lodash/fp'
+import { useState } from 'react'
+import { Ajax } from 'src/libs/ajax'
+import { reportError, withErrorReportingInModal } from 'src/libs/error'
+import { useCancellation, useOnMount } from 'src/libs/react-utils'
+import { getUser } from 'src/libs/state'
+import * as Utils from 'src/libs/utils'
+import { BaseWorkspace, isGoogleWorkspace, WorkspaceInfo } from 'src/libs/workspace-utils'
+import { isResourceDeletable } from 'src/pages/workspaces/workspace/analysis/runtime-utils'
+
+
+interface DeleteWorkspaceModalLeoApp {
+  appName: string
+  status: string
+  cloudContext: any
```

Comment:
This is here because the DeleteWorkspaceModal code references it, not sure if there's a better type than `any` to introduce here.

Reply:
I'm not seeing a TypeScript definition of this yet in IA code.

Reply:
see my comment where this type is used.... also, I recommend chatting with IA team to assist with a better type for cloudContext.  Getting some cross-team pairing on stuff like this to get good types is really important.